### ISSUE TITLE
Cant see Chomecast device - Only seeing partial pop up box

### ISSUE THREAD

**Issue by githoward** [acidhax/chromecast-audio-stream#96]
![chromeaudiostr](https://user-images.githubusercontent.com/33431890/32459889-069bdf3c-c2ff-11e7-833f-c95e9c32cef3.jpg)

See image.
Using my main computer (Win 7 Pro 64 Bit); On right click, I only see part of the box (close). I cannot see my Chomecast device.  If i try Chromecast Audio Stream on my laptop, I am able to see my Chromecast device and it works great.

**Comment by githoward** [comment]
I found the answer - Run Chromecast Audio Stream from a Desktop Shortcut. In Compatibility tab, in Privelege Level Section, check Run this program as Administrator

**Comment by Adiventure** [comment]
I'm seeing the same issue, tried running from the exe with administrator, as well as from a shortcut on desktop.

**Comment by githoward** [comment]
Did you do the Compatibility tab > Privelege Level Section and check Run this program as Administrator

**Comment by Adiventure** [comment]
Yeah, it's set to run as administrator, and pops up an UAC prompt whenever launched.

**Comment by nononolimits** [comment]
I'm having the same issue. Worked well a few days ago. I think problems started since latest windows updates.
I also tried to set "run as admin", but that doesn't help for me either.

**Comment by nononolimits** [comment]
Never mind. It's working again now. Don't know how or why.

**Comment by epicnessinaperson** [comment]
I am having the exact same problem as OP, after I run the program it looks exactly as depicted in the picture above, there is absolutely no option to connect to chromecast, just a close button. I have the program files in a folder on the desktop and a shortcut to the .exe on the desktop that I am using to run the program. I have enabled administrator privileges in like three places under the properties of the shortcut. I've even tried compatibility settings for windows 7 and 8 even though I'm on windows 10. This is my first time trying to install and run this program and I really hope to be able to find a solution to this problem because it would be rather convenient for me to have a program to directly stream audio to chromecast. Oh, and I tried to install the "Microsoft Visual C++ 2010 Redistributable Package" and it said that a newer version was already installed so I would assume that's not the problem...

**Comment by nononolimits** [comment]
For me, it just started working again after a simple reboot. Or maybe it was because I've restarted my Chromecast Audio device as well at that same time (I restarted it even though I could find it in my network from other devices). 
Haven't had any issues again since then.

Are you sure your Chromecast is on your network at the moment? Can you find it in Youtube on your smartphone, for example?

**Comment by epicnessinaperson** [comment]
Wait, does it need to be a chromecast audio device? I simply have the original chromecast, not the ultra, nor the audio. But yeah I mean I use it all the time, so as for whether it's on the same network I can guarantee that it definitely is, I just used it this morning.

**Comment by epicnessinaperson** [comment]
Just to add to and clarify the previous point, I am currently able to stream to chomecast from the chrome browser on the same computer I am attempting to install this chromecast audio software.

**Comment by MaCXyLo** [comment]
similar issue - found a workaround:

Install Microsoft Visual C++ 2010 Redistributable Package (x86):
https://www.microsoft.com/en-us/download/details.aspx?id=5555

Install NPM (Node Package Manager):
https://nodejs.org/en/download/

Install git:
https://git-scm.com/download/win

Download the latest version from chromecast audio stream:
git clone https://github.com/acidhax/chromecast-audio-stream.git
The release 0.3 isn't up2date.

npm install -g grunt-cli

Disable all network adapters that are not necessary:
Control Panel\Network and Internet\Network and Sharing Center\Network Connections

Change directory to the cloned repository, 
install and execute the program from the powershell as administrator:
npm install
grunt

after that, the program should find the chromecast without any problem.
seems to cache the ip and name of the device.
now you can activate the network adapters.

ideas to fix this issue:

workarounds:
Let the user choose the network adapter.
Let the user choose an ip address (fuckin hell).

final solution:
work on a proper detection of the network adapter.

**Comment by itsnottakenme** [comment]
Hmm for me this only happens when the menu is at the top of the screen. The problem is the menu only appears above the icon so if it's at the top of the screen the menu gets truncated. If I move the icon from the notification area to the collapsed notification area then I can see the whole menu pop up without it going off the screen
